KRKNWK-19540: Update RS/FEM projects related documentation (#17845)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/fem/fem_nrf21540_gpio.rst

   The state of the remaining control pins should be set in other ways and according to `nRF21540 Product Specification`_.

> #. On nRF53 devices, you must also apply the same devicetree node mentioned in step 1 to the network core  using sysbuild build system. For more information, see the :ref:`Migrating to sysbuild <child_parent_to_sysbuild_migration>` userguide.
>    To do so, apply the overlay to the correct network core child image by creating an overlay file named :file:`sysbuild/*childImageName*/boards/nrf5340dk_nrf5340_cpunet.overlay` in your application directory, for example :file:`sysbuild/ipc_radio/boards/nrf5340dk_nrf5340_cpunet.overlay`.

Reviewer: ```suggestion
#. On nRF53 devices, you must also apply the same devicetree node mentioned in **Step 1** to the network core  using sysbuild.
   To apply the overlay to the correct network core child image, create an overlay file named :file:`sysbuild/*childImageName*/boards/nrf5340dk_nrf5340_cpunet.overlay` in your application directory, for example :file:`sysbuild/ipc_radio/boards/nrf5340dk_nrf5340_cpunet.overlay`.
   For more information, see the :ref:`Migrating to sysbuild <child_parent_to_sysbuild_migration>` userguide.
```

>    The ``*childImageName*`` default value is set to ``ipc_radio`` :ref: `ipc_radio`:

Reviewer: ```suggestion
   The *childImageName* default value is set to ``ipc_radio``:
```

   *  ``ipc_radio`` represent all applications having support for combination of both 802.15.4 and Bluetooth. Application can be configured base on sysbuild configs:
      * ``SB_CONFIG_NETCORE_IPC_RADIO=y`` for applications having support for 802.15.4, but not for Bluetooth.
      * ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC=y`` for application having support for Bluetooth, but not for 802.15.4.

>       * ``SB_CONFIG_NETCORE_IPC_RADIO=y`` and ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC=y`` for multiprotocol applications having support for both 802.15.4 and Bluetooth.

Reviewer: Same fixes as above to all edited parts.

   The state of the remaining control pins should be set in other ways and according to `nRF21540 Product Specification`_.

#. On nRF53 devices, you must also apply the same devicetree node mentioned in **Step 1** to the network core  using sysbuild.
   To apply the overlay to the correct network core child image, create an overlay file named :file:`sysbuild/*childImageName*/boards/nrf5340dk_nrf5340_cpunet.overlay` in your application directory, for example :file:`sysbuild/ipc_radio/boards/nrf5340dk_nrf5340_cpunet.overlay`.

>    For more information, see the :ref:`Migrating to sysbuild <child_parent_to_sysbuild_migration>` userguide.

Reviewer 2: ```suggestion
   For more information, see the :ref:`Migrating to sysbuild <child_parent_to_sysbuild_migration>` page.
```
